apps/agentic-chat/src/lib/langchain.ts

```diff
@@ -46,13 +47,23 @@ const workflow = new StateGraph(MessagesAnnotation)
   .addEdge('tools', 'agent')
   .addConditionalEdges('agent', shouldContinue);
 
+// Adds persistence
+const checkpointer = new MemorySaver();
```

Comment:
We were previously missing this - this will create an in-memory pointer for persistence (i.e unique to current user's tab). Nothing fancy for now but allows back and forth convos.